app/controllers/api/base_controller.rb

```diff
@@ -4,6 +4,7 @@
 
 module Api
   class BaseController < ActionController::Metal
+    include RawParams
```

Comment:
What's the long term strategy here? Should we be looking to phase this out and use the Rails 5+ approach to param handling?